diktat-gradle-plugin/src/functionalTest/kotlin/org/cqfn/diktat/plugin/gradle/DiktatGradlePluginFunctionalTest.kt

```diff
@@ -39,6 +40,32 @@ class DiktatGradlePluginFunctionalTest {
         )
     }
 
+    /*
+        * Should fix output here
+     */
+    @Test
+    fun `should have json reporter files`() {
+        buildFile.appendText(
+            """${System.lineSeparator()}
+                diktat {
+                    inputs = files("src/**/*.kt")
+                    reporterType = "json"
+                    output = "test.txt"
+                }
+            """.trimIndent()
+        )
+        val result = runDiktat(testProjectDir, shouldSucceed = false)
+
+        val diktatCheckBuildResult = result.task(":$DIKTAT_CHECK_TASK")
+        requireNotNull(diktatCheckBuildResult)
+        Assertions.assertTrue(testProjectDir.root.walkTopDown().filter { it.name == "test.txt" }.firstOrNull() != null)
```

Comment:
`assertTrue(file != null)` later instead of looking for it twice?

Reply:
Fixed